pkgs/applications/version-management/github-desktop/default.nix

```diff
@@ -45,7 +46,7 @@ stdenv.mkDerivation rec {
     alsa-lib
     cups
     mesa
-    openssl
+    openssl_1_1
```

Comment:
🤔

Reply:
Otherwise building fails with
```
auto-patchelf: 2 dependencies could not be satisfied
> error: auto-patchelf could not satisfy dependency libssl.so.1.1 wanted by /nix/store/syjc2z3p6k2cnqm11lqfm597mf82akrs-github-desktop-3.2.1/opt/resources/app/git/libexec/git-core/git-imap-send
> error: auto-patchelf could not satisfy dependency libcrypto.so.1.1 wanted by /nix/store/syjc2z3p6k2cnqm11lqfm597mf82akrs-github-desktop-3.2.1/opt/resources/app/git/libexec/git-core/git-imap-send
> auto-patchelf failed to find all the required dependencies.
```

I don't really understand why it builds on Hydra though, maybe an impurity?

Reply:
`openssl` is actually `openssl_1_1` in this package, see https://github.com/NixOS/nixpkgs/blob/423204a9821710b7f85af25aefea83aaef1ed0f3/pkgs/top-level/all-packages.nix#L3341-L3344.

Reply:
Oh alright I'll remove this change then, sorry.